src/cdk/scrolling/virtual-scroll-viewport.ts

```diff
@@ -67,11 +69,12 @@ const SCROLL_SCHEDULER =
   providers: [
     {
       provide: CdkScrollable,
-      useExisting: CdkVirtualScrollViewport,
+      useFactory: (scrollViewport: CdkVirtualScrollViewport) => scrollViewport.scrollable,
```

Comment:
This feels a little bit odd to me. Maybe instead it should be something like this?
`(virtualScrollable: CdkVirtualScrollable|null, viewport: CdkVirtualScrollViewport) => virtualScrollable || viewport`